### ISSUE TITLE
Tooltip doesn't close when wrapping a disabled button in Chrome

### ISSUE THREAD

**Issue by alinchican** [salesforce/design-system-react#1726]
Hello! 

I'm having issues with disabled buttons inside tooltips, the tooltip content isn't hidden on mouse out on Chrome (version 71). On Firefox (version 64) everything seems ok. You can view an example [here](https://codesandbox.io/s/pkrx5w9j5q).

Tested using:

- Chrome 71
- @salesforce-ux/design-system: 2.8.1
- @salesforce/design-system-react: 0.9.4
- react: 16.7.0

Am I missing something? I've searched for a related issue but couldn't find anything. 

Thank you!

**Comment by interactivellama** [comment]
It appears that disabled button in Chromium do not have onMouseLeave events.
https://github.com/facebook/react/issues/4251

You might try controlling it with a parent. https://react.lightningdesignsystem.com/components/tooltips/#prop-isOpen

**Comment by interactivellama** [comment]
I did a little more research. As stated above, the browser isn't firing the events that React needs. I looked at React Bootstrap's implementation, and they've hacked a solution of creating a duplicate overlay on top of disabled buttons in order to allow this. https://github.com/react-bootstrap/react-bootstrap/blob/04184495ce111d9e0a9d100cec8f8b8122015ccf/src/OverlayTrigger.js#L173

One thing to keep in mind though is that disabled buttons are not focusable and therefore the tooltip will never show for non-mouse users such as screen reader users and therefore tooltips on disabled items are discouraged in the Lightning Design System. If there is room in the layout, I would recommend an info icon next to the disabled button. I would recommend iterating on a solution with a designer if are you working with a Salesforce product?

Are you working with Dustin McCormick?
